apps.momits.Electrip.json

```diff
@@ -0,0 +1,142 @@
+{
+    "app-id": "apps.momits.Electrip",
```

Comment:
This isn't a valid app-id. https://momits.apps isn't valid.

Reply:
You are right. Currently I do not own a domain by myself. Should I use ```net.sourceforge.electrip```?

Reply:
Yup that will do.